.github/workflows/ci.yml

```diff
@@ -76,6 +76,8 @@ jobs:
           noPrefix: true
           awsAccessKeyId: ${{ secrets.AWS_ACCESS_KEY_ID }}
           awsSecretAccessKey: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          githubToken: ${{ secrets.GITHUB_TOKEN }}
+          deployRunUrl: https://portal-report.concord.org/__deployPath__/
```

Comment:
In VSCode, there was a lint error for this `with` section: "Missing required input `prefix`". Even though `noPrefix` is present and set to true, `s3-deploy-action` still requires `prefix` ([see here](https://github.com/concord-consortium/s3-deploy-action/blob/main/action.yml#L14-L16)).

Seems like this is working without `prefix`, but should we add it? Or update `s3-deploy-action` so `prefix` isn't required?

Reply:
I would be good to update the s3-deploy-action so prefix isn't required.